library/src/morpheus-master/sass/modules/connection-manager/_base.scss

```diff
@@ -334,8 +334,3 @@
 		height: 375px;
   }
 }
-
-.modal-backdrop {
-  background-color: #FFFFFF !important;
-  opacity: 0.4 !important;
-}
```

Comment:
Removing these bootstrap overrides completely after discussing with Shawn. Previous commit I had updated the bgcolor with a custom property, but it isn't really necessary at all.